raiden-dapp/src/views/GeneralDialog.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="general-screen">
+    <v-row class="general-screen__header" no-gutters>
+      <div class="general-screen__header__content">
+        <div class="general-screen__header__content__back">
+          <v-btn
+            height="40px"
+            text
+            icon
+            width="40px"
+            @click="onGeneralBackClicked()"
+          >
+            <v-img :src="require('../assets/back_arrow.svg')" max-width="34px">
+            </v-img>
+          </v-btn>
+        </div>
+        <div class="general-screen__header__content__title">
+          {{ $route.meta.title }}
+        </div>
+      </div>
+    </v-row>
+    <router-view></router-view>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Mixins } from 'vue-property-decorator';
+import NavigationMixin from '@/mixins/navigation-mixin';
+
+@Component({})
+export default class GeneralDialog extends Mixins(NavigationMixin) {}
+</script>
+
+<style scoped lang="scss">
+@import '../scss/mixins';
+@import '../scss/fonts';
+
+.general-screen {
+  background-color: black; /* CHANGE */
+  border-radius: 10px;
+  height: 844px;
+  position: absolute;
+  width: 620px;
+  z-index: 90000;
```

Comment:
Not big of a deal more of a side note: If a lower z-index like 2 or 3 is sufficient then it could be used instead. Otherwise it can get tricky to find out later on if we want to have a component above the general dialog what the correct z-index might be